chainweb.cabal

```diff
@@ -265,6 +267,33 @@ library
         , Utils.Logging.Config
         , Utils.Logging.Trace
 
+        -- tools
+        , Tools.Chain2Gexf
+        , Tools.CwTool
+        , Tools.Ea
+        , Tools.Gexf
+        , Tools.RunNodes
+        , Tools.Standalone
+        , Tools.Standalone.Chainweb
+        , Tools.Standalone.Utils
+        , Tools.TXG
+        , Tools.TXG.Repl
+        , Tools.TXG.ReplInternals
+        , Tools.TXG.Simulate.Contracts.CoinContract
+        , Tools.TXG.Simulate.Contracts.Common
+        , Tools.TXG.Simulate.Contracts.HelloWorld
+        , Tools.TXG.Simulate.Contracts.SimplePayments
+        , Tools.TXG.Simulate.Utils
+        , Tools.TXG.Types
+        , Tools.Tests.MultiNode
+        , Tools.Tests.Network.X509.SelfSigned.Test
+        , Tools.Tests.Orphans.External
+        , Tools.Tests.Orphans.Internal
+        , Tools.Tests.P2P.Peer.BootstrapConfig
+        , Tools.Tests.SlowTests
+        , Tools.Tests.Utils
+        , Tools.TxStream
```

Comment:
Is this really needed?

I think, this generally goes into the wrong direction. The chainweb library is already far too big. It takes a long time to build and becomes increasingly harder to maintain.